File: TelerikUIExamples/TelerikUIExamples/DocsCustomCell.m
//
//  DocsCustomCell.m
//  TelerikUIExamples
//
//  Copyright © 2016 Telerik. All rights reserved.
//

#import "DocsCustomCell.h"
#import <TelerikUI/TelerikUI.h>

// >> customization-customcell
@implementation DocsCustomCell

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
    }
    return self;
}

- (void)updateVisuals
{
    [super updateVisuals];
    
    if (self.state & TKCalendarDayStateToday) {
        self.label.textColor = [UIColor redColor];
    }
    else {
        self.label.textColor = [UIColor blueColor];
    }
}

@end
// << customization-customcell

File: TelerikUIExamplesInSwift/TelerikUIExamplesInSwift/DocsCustomCell.swift
//
//  DocsCustomCell.swift
//  TelerikUIExamplesInSwift
//  Copyright © 2016 Telerik. All rights reserved.
//

import Foundation

// >> customization-customcell-swift
class DocsCustomCell: TKCalendarDayCell {
    override func updateVisuals() {
        super.updateVisuals()
        
        if self.state.rawValue & TKCalendarDayState.Today.rawValue != 0 {
            self.label.textColor = UIColor.redColor()
        }
        else {
            self.label.textColor = UIColor.blueColor()
        }
    }
}
// << customization-customcell-swift